install_config/persistent_storage/topics/glusterfs_storage_sizing.adoc

```diff
@@ -0,0 +1,25 @@
+Every GlusterFS cluster must be sized based on the needs of the anticipated
+applications that will use its storage. Some additional things to consider are:
+
+- For each {gluster-native} or {gluster-external} cluster the default behavior
+is to create GlusterFS volumes with three-way replication. As such, the total
+storage to plan for should be the desired capacity times three.
+  ** As an example, each heketi instance creates a `heketidbstorage` volume
```

Comment:
s/heketi/Heketi

Reply:
heketi is intentionally all lowercase. That is the official stylization of the name.